pycds/__init__.py

```diff
@@ -16,7 +17,7 @@
 from geoalchemy2 import Geometry
 
 
-Base = declarative_base()
+Base = declarative_base(metadata=MetaData(schema='crmp'))
 metadata = Base.metadata
```

Comment:
Oddly, one cannot factor metadata out of the `Base =` expression and avoid the apparently repetitive next line. Tests fail left and right. What comes out is not just what goes in.